### DISCUSSION TITLE
50%+1 attack warning

### DISCUSSION THREAD

**Discussion by iXavierLiu** [Chia-Network/chia-blockchain#3887] [Q&A]
hpool power is close to 50% of the whole network, If it reaches 50% + 1, will it impact the existing system, and does Chia have a way to resist attacks?
-
hpool的算力已经接近整个网络的50％，如果达到50％+ 1，是否会对现有体系造成影响，Chia是否有办法抵御攻击？
-
hpoolの計算能力はネットワーク全体の50％に近いですが、50％+ 1に達すると、既存のシステムに影響しますか？Chiaには攻撃に抵抗する方法がありますか？
-

**Comment by SackofFluor** [comment]
thanks for the wearning, but technically we cant do anything about it?

**Comment by jackalchenxu** [comment]
we just wait. when XCH price dropped,  there is no 50% issue.
